### DISCUSSION TITLE
Live server

### DISCUSSION THREAD

**Discussion by abuhabban** [coder/code-server#3994] [General] [answered]
How can i go live. I already installed live server when i start live server it says server is running in 5500 port but i Have deployed vs code in heroku.
Please help

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
For what do you use Live Server for? If you have ports like 4000, use this:

`https://{{URL}}/proxy/4000/`

Don't forget to add the slash at the end. And change {{URL}} to your Heroku URL

This is same for all ports.

**Comment by jsjoeio** [comment]
Have you followed the [official Heroku guide](https://github.com/cdr/deploy-code-server/blob/main/guides/heroku.md)? Maybe you could share a video and elaborate on what's not working? 

That will help us help you faster :)
